src/io/pti.jl

```diff
@@ -943,4 +943,744 @@ function _populate_defaults!(data::Dict)
+    
+    println(io, "0 / END OF VOLTAGE SOURCE CONVERTER DATA, BEGIN IMPEDANCE CORRECTION DATA")
+
+    Memento.warn(_LOGGER, string("Export IMPEDANCE CORRECTION data is not yet supported"))
+    
+    println(io, "0 / END OF IMPEDANCE CORRECTION DATA, BEGIN MULTI-TERMINAL DC DATA")
+    
+    Memento.warn(_LOGGER, string("Export MULTI-TERMINAL DC data is not yet supported"))
+    
+    println(io, "0 / END OF MULTI-TERMINAL DC DATA, BEGIN MULTI-SECTION LINE DATA")
+    
+    Memento.warn(_LOGGER, string("Export MULTI-SECTION data is not yet supported"))
+    
+    println(io, "0 / END OF MULTI-SECTION LINE DATA, BEGIN ZONE DATA")
+
+    # Zone Data
+    if haskey(data, "zone") 
+        for (_, zone) in sort(data["zone"], by = (x) -> parse(Int64, x))
+            # Get Dict in a PSSE way and print it
+            psse_comp = _pm2psse_zone(zone)
+            _print_pti_str(io, psse_comp, _pti_dtypes["ZONE"])
+        end
+    else
+        Memento.warn(_LOGGER, string("Skipping ZONE data because it does not found"))
+    end
+
+
+    println(io, "0 / END OF ZONE DATA, BEGIN INTER-AREA TRANSFER DATA")
+
+    Memento.warn(_LOGGER, string("Export inter-area transfer data is not yet supported"))
+    # Inter Area Data
+    # Since parse_pti only parse "ptran", "trid" it cannot be replicated 
+    # if haskey(data, "inter-area transfer") 
+    #     for (_, interarea) in sort(data["inter-area transfer"], by = (x) -> parse(Int64, x))
+    #         # Get Dict in a PSSE way and print it
+    #         psse_comp = _pm2psse_interarea(interarea)
+    #         _print_pti_str(io, psse_comp, _pti_dtypes["INTER-AREA TRANSFER"])
+    #     end
+    # end
+
+    println(io, "0 / END OF INTER-AREA TRANSFER DATA, BEGIN OWNER DATA")
+
+    # Owner Data
+    if haskey(data, "owner") 
+        for (_, owner) in sort(data["owner"], by = (x) -> parse(Int64, x))
+            # Get Dict in a PSSE way and print it
+            psse_comp = _pm2psse_owner(owner)
+            _print_pti_str(io, psse_comp, _pti_dtypes["OWNER"])
+        end
+    else
+        Memento.warn(_LOGGER, string("Skipping OWNER data because it does not found"))
+    end
+
+    println(io, "0 / END OF OWNER DATA, BEGIN FACTS CONTROL DEVICE DATA")
+
+    Memento.warn(_LOGGER, string("Export FACTS data is not yet supported"))
+
+    println(io, "0 / END OF FACTS CONTROL DEVICE DATA, BEGIN SWITCHED SHUNT DATA")
+
+    # Switched Shunt
+    for (_, shunt) in sort(data["shunt"], by = (x) -> parse(Int64, x))
+        # Skip Fixed Shunts
+        if shunt["source_id"][1] != "switched shunt"
+            continue
+        end
+        
+        # Get Dict in a PSSE way
+        psse_comp = _pm2psse_switched_shunt(shunt)
+
+        # Print it in the file
+        _print_pti_str(io, psse_comp, _pti_dtypes["SWITCHED SHUNT"])
+    end
+
+    println(io, "0 /END OF SWITCHED SHUNT DATA, BEGIN GNE DEVICE DATA")
+
+    Memento.warn(_LOGGER, string("Export GNE data is not yet supported"))
```

Comment:
I would only warn if the data contains the key that is associated with `import_all=true` and the data in there is not empty.  The message should say "not yet supported by the export_pti function"

Reply:
Solved at the top of the function

Reply:
I didn't see it, can you clarify where this is happening?

Reply:
Sorry for the mess. It is happenning [here](https://github.com/lanl-ansi/PowerModels.jl/pull/753/commits/043ec834f90e996db76f12764efec643a2c256ed#diff-8eaa62afd9f02012047d607fc11deb5fc6d076a155ecf49c2ba89625666653afR1018)